src/server.ts

```diff
@@ -1,7 +1,10 @@
-import express from 'express'
-import indexController from '../src/controllers/index'
-export const app = express()
+import net from 'node:net'
 
-app.get('/', indexController)
+const port: number = 3000
+const host: string = 'localhost'
 
-app.listen(3000)
+const server = net.createServer((socket: net.Socket): void => {})
+
+server.listen(port, host, (): void => {
+  console.log(`TCP server has established a connection at http://${host}:${port}/.`)
```

Comment:
Indeed, it is important not to test code we don't control. 

But what about this console log statement? 

I suggest writing a test for this!

Reply:
The file this references was added to its own 'server' directory and can be found in `src/server/server.ts`, and the mock test for the console statement has been created and can be found in `tests/console/console.ts`. Thank you for the help! :)